test/ModalTriggerSpec.js

```diff
@@ -101,4 +101,19 @@ describe('ModalTrigger', function() {
 
     contextSpy.calledWith('value').should.be.true;
   });
+
+  it('Should pass children refs through. Prevent future regressions.', function () {
```

Comment:
I think the "Prevent future regressions." part is superfluous, it's what tests do. :wink: